pkg/apis/wksprovider/machine/os/os.go

```diff
@@ -281,6 +284,7 @@ func (o OS) CreateSeedNodeSetupPlan(params SeedNodeParams) (*plan.Plan, error) {
 			Namespace:             object.String(params.Namespace),
 			NodeName:              cfg.HostnameOverride,
 			ExtraAPIServerArgs:    apiServerArgs,
+			ServiceCIDRBlock:      params.ServicesCIDRBlocks[0],
```

Comment:
Do we know this slice is always of length one?
If so, I think you should comment where that guarantee comes from.
If not, this function needs to be more defensive.

Reply:
While testing with a cluster.yaml that had no CIDR blocks defined or multiple, I see there is validation for the fields in [pkg.specs.validation.go](https://github.com/weaveworks/wksctl/blob/75f89d592673db089ef222f9f23fc1cd9694b7fe/pkg/specs/validation.go#L87). In both cases the error is caught and it prints:
```
➜  gce git:(WKP-587/set-pod-and-service-cidr-blocks) ✗ wksctl apply   
ERRO[2020-06-04T19:30:11+02:00] cluster.spec.clusterNetwork.services.cidrBlocks: Invalid value: []string(nil): CIDR blocks must contain exactly one IP range 
ERRO[2020-06-04T19:30:11+02:00] cluster.spec.clusterNetwork.pods.cidrBlocks: Invalid value: []string{"192.168.0.0/16", "172.20.0.0/23"}: CIDR blocks must contain exactly one IP range 
FATA[2020-06-04T19:30:11+02:00] Error parsing manifest: cluster.yaml failed validation, use --skip-validation to force the operation
```
The `--skip-validation` flag seems to be broken currently:
```
➜  gce git:(WKP-587/set-pod-and-service-cidr-blocks) ✗ wksctl apply --skip-validation
Error: unknown flag: --skip-validation
Usage:
...
```

Added a comment to the validation function and the kubeadm [Networking](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#Networking) struct.